### DISCUSSION TITLE
I need help with my bot

### DISCUSSION THREAD

**Discussion by ghost** [PrismarineJS/mineflayer#2659] [Q&A]
I got the cloop working all i need is so for a ingame command if i do !deop Dream it deops the specific player so in that case its dream 
this is my code   

 bot.on('chat', (username, message) => {
  if (message.startsWith('!deop')){
//command block command
    setInterval(function() { runInCore('execute run deop'), 1})
    
     // Run with the sender and the command itself
    
  }
I have been stuck on this for a about 2 days since im terrible and new to node.js

**Comment by amoraschi** [comment]
What's a cloop

**Reply by ghost** [reply]
A Command Block Loop

**Comment by ghost** [comment]
You could either:
- Make a chat pattern.
 * Mess around with regex and switch function.
- Use bot.on('message').
 * Mess around with array and switch function.


* Imo I would use the first option as its easier.
